unison-src/transcripts-using-base/utf8.output.md

```diff
@@ -5,8 +5,8 @@ Unison has function for converting between `Text` and a UTF-8 `Bytes` encoding o
 ```ucm
 .> find Utf8
 
-  1. Text.fromUtf8 : Bytes ->{Exception} Text
-  2. builtin.Text.toUtf8 : Text -> Bytes
+  1. builtin.Text.toUtf8 : Text -> Bytes
+  2. Text.fromUtf8 : Bytes ->{Exception} Text
```

Comment:
Just different sorting. Honestly not sure what the old sorting was here... was it reversed or what?

Reply:
Weird, idk